src/configuration.ts

```diff
@@ -660,3 +660,24 @@ export function isFileExcluded(filepath: string, configFile?: IConfigurationFile
     const fullPath = path.resolve(filepath);
     return configFile.linterOptions.exclude.some(pattern => new Minimatch(pattern).match(fullPath));
 }
+
+export function stringifyConfiguration(configFile: IConfigurationFile) {
```

Comment:
re: testing. I think a couple unit tests for this function should be sufficient; we don't really need to verify this feature through the CLI (but you can keep the tests you have added thus far)